VideoWeb/VideoWeb/ClientApp/src/app/shared/select-media-devices/select-media-devices.component.scss

```diff
@@ -45,12 +45,6 @@
 .wr-h {
   margin-left: 25px;
 }
-/*mirrored video*/
-#preferredCameraStream {
-  transform: rotateY(180deg);
-  -webkit-transform: rotateY(180deg); /* Safari and Chrome */
-  -moz-transform: rotateY(180deg); /* Firefox */
-}
```

Comment:
Does this still get mirrored another way?

Reply:
yes, via the video filter